tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "preserve"
```

Comment:
why was this changed to preserve ?

Reply:
I was having problems with my package.json probably changed it

Reply:
will change it